File: 2298-count-integers-with-even-digit-sum/2298-count-integers-with-even-digit-sum.py
class Solution:
    def countEven(self, num: int) -> int:
        j=0
        for i  in range(2,num+1):
            s=0
            while(i):
                r=i%2
                s+=r
                i//=10
            if(s%2==0):
                j+=1
        return j
        
              
        

        

File: 2298-count-integers-with-even-digit-sum/2298-count-integers-with-even-digit-sum.cpp
class Solution {
public:
    int countEven(int num) {
        long long s = 0;
        int i = 2;
        while (i <= num) {
            int z = 0;
            int k=i;
            while (k) {
                int r=k%10;


                z+=r;
                k/= 10;
            }
            if (z % 2==0 ) {
                s++;
            }
            i++;
        }
        return s;
    }
};